code/modules/mob/living/life.dm

```diff
@@ -201,6 +204,16 @@
 			AdjustEarDamage(-0.05)
 			AdjustEarDeaf(-1)
 
+/mob/living/proc/handle_stasis()
+	// Handle side effects from stasis
+	if(in_stasis)
```

Comment:
Why does this check `in_stasis` if it can only be called when `is_stasis` is true?